pages/common/bundle.md

```diff
@@ -11,9 +11,21 @@
 
 `bundle update`
 
-- Update one specific gem defined in the `Gemfile`:
+- Update one/multiple specific gem(s) defined in the `Gemfile`:
 
-`bundle update --source {{gemname}}`
+`bundle update {{gemname}} {{gemname}}`
+
+- Update one/multiple specific gems(s) defined in the `Gemfile` but only to the next patch version:
```

Comment:
Perhaps this will read better?
```suggestion
- Update one or more specific gems(s) defined in the `Gemfile` but only to the next patch version:
```